bb/bb.go

```diff
@@ -336,13 +327,20 @@ func oneCmd(c Command) {
 		}
 	}
 	initMap += "\n\t\"" + c.CmdName + "\":" + c.CmdName + "Init,"
+	// In the bb case, the commands are built. In some cases, we want to
+	// specify init= for a u-root command on boot. Hence, it now makes sense
```

Comment:
Good question. That may explain why init= was not working for me? Supper-annoying if we can not over-ride /init! SAD!

Change 'for' to 'to be'